marpa/lib/Guacamole.pm

```diff
@@ -13,8 +13,36 @@ StatementSeq ::= Statement
                | Statement Semicolon
                | Statement Semicolon StatementSeq
 
-Statement ::= Expression
+Statement ::= NonBraceExpression StatementModifier
+            | NonBraceExpression
             | Block
+            | Condition
+
+StatementModifier ::= ConditionIfPostfixExpr
+                    | ConditionUnlessPostfixExpr
+                    | ConditionWhilePostfixExpr
+                    | ConditionUntilPostfixExpr
+                    | ConditionForPostfixExpr
+                    | ConditionForeachPostfixExpr
+                    | ConditionWhenPostfixExpr
+
+Condition ::= ConditionIfExpr ConditionElsifExpr ConditionElseExpr
```

Comment:
It is possible to have multiple `elsif` branches in the same statement.

I _think_ that marpa supports empty branches, so it can be modelled with something like this:
```
Condition ::= ConditionIfExpr ConditionIfTail | ConditionUnlessExpr
ConditionIfTail ::= # Can be empty
  | ConditionElsifExpr ConditionIfTail
  | ConditionElseExpr
```

Reply:
I think we can solve it by having an ElsIf allow another Elsif infinitely, and the If could have ElsIf and Else. I'll try it out.

Reply:
I added a commit that supports multiple `elsif` statements.